.circleci/unittest/linux/scripts/install.sh

```diff
@@ -23,10 +23,13 @@ else
     version="$(python -c "print('.'.join(\"${CUDA_VERSION}\".split('.')[:2]))")"
 
     cuda_toolkit_pckg="cudatoolkit"
+    cudatoolkit=""
     if [[ "$CU_VERSION" == cu116 ]]; then
         cuda_toolkit_pckg="cuda"
+        cudatoolkit="-c nvidia cuda=11.6 atalman::pytorch-cuda=1.0=cuda11.6"
```

Comment:
Why do you need `cuda=11.6` package here? 
```suggestion
        cudatoolkit="-c nvidia atalman::pytorch-cuda=1.0=cuda11.6"
```

Reply:
Basically metapackage just sets constraints. Selecting only metapackage does not work

```
conda install -c nvidia -c pytorch-nightly pytorch-cuda=1.0=cuda11.6
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/atalman/anaconda3/envs/mpy3.91

  added / updated specs:
    - pytorch-cuda==1.0=cuda11.6


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pytorch-cuda-1.0           |         cuda11.6           6 KB  pytorch-nightly
    ------------------------------------------------------------
                                           Total:           6 KB

The following NEW packages will be INSTALLED:

  pytorch-cuda       pytorch-nightly/noarch::pytorch-cuda-1.0-cuda11.6


Proceed ([y]/n)? n
```